### PR TITLE
contributor guidelines draft

### PR DESCRIPTION
**Pull request by gdicecco**
I added a draft of some text to populate the Contributor Guidelines page of the readthedocs documentation. To review, follow the instructions in `docs/README` to build the documentation locally and view in your browser. 

I'm open to suggestions on content/formatting/etc.!

### CODE REVIEW

File: docs/contribute.rst [hunk 1]
```diff
@@ -2,4 +2,31 @@
...
+1. `climakitae <https://github.com/cal-adapt/climakitae>`_ (Climate Kit – Analytics Engine, get it?)
+
+	Building primarily on `xarray <https://docs.xarray.dev/en/stable/>`_ and the whole `Pangeo 	<https://pangeo.io/>`_ ecosystem, climakitae primarily offers tools that are:
+
+	* Meant to be run inside a Jupyter notebook (see below)
+	* Specific to the data for the `State of California <https://analytics.cal-adapt.org/data/>`_ that has 	been downscaled based on CMIP6 climate models.
+
+2. The `cae-notebooks <https://github.com/cal-adapt/cae-notebooks>`_ repository 
+
+	`Jupyter <https://jupyter.org/>`_ notebooks in python which:
```
Comment (claalmve) [inline docs/contribute.rst:18]:
```suggestion
	`Jupyter <https://jupyter.org/>`_ notebooks in Python which: 
```

File: docs/contribute.rst [hunk 2]
```diff
@@ -2,4 +2,31 @@
...
+2. The `cae-notebooks <https://github.com/cal-adapt/cae-notebooks>`_ repository 
+
+	`Jupyter <https://jupyter.org/>`_ notebooks in python which: 
+
+	* Highlight a particular climakitae tool, or
+	* Do a specific analysis for an example use-case or application of the data
+
+We welcome contributions to either repository.
+
+Contributions to climakitae that we would welcome could involve a performance improvement, something which makes any of the tools more generally applicable, or adds a feature to handle a commonly-requested operation (without duplicating anything that already exists in the Pangeo ecosystem).
```
Comment (claalmve) [inline docs/contribute.rst:25]:
```suggestion
Welcomed contributions could involve a performance improvement, something which makes any of the tools more generally applicable, or adds a feature to handle a commonly-requested operation (without duplicating anything that already exists in the Pangeo ecosystem).
```

File: docs/contribute.rst [hunk 3]
```diff
@@ -2,4 +2,31 @@
 Contribution Guidelines
 **********************
 
-How to contribute to the code base for external developers. Filler. 
\ No newline at end of file
+This is an open-source project, and we welcome contributions from the community. 
+
+The Analytics Engine consists of two separate open-source repositories (hosted on GitHub) that you may contribute to.
+
+1. `climakitae <https://github.com/cal-adapt/climakitae>`_ (Climate Kit – Analytics Engine, get it?)
```
Comment (vicford) [inline docs/contribute.rst:9]:
```suggestion
1. The `climakitae <https://github.com/cal-adapt/climakitae>`_ Python library:
```

File: docs/contribute.rst [hunk 4]
```diff
@@ -2,4 +2,31 @@
 Contribution Guidelines
 **********************
 
-How to contribute to the code base for external developers. Filler. 
\ No newline at end of file
+This is an open-source project, and we welcome contributions from the community.
```
Comment (vicford) [inline docs/contribute.rst:5]:
```suggestion
This is an open-source project, and all contributions, including new feature development, documentation, bug reports, bug fixes, and other ideas are welcome. 
```

File: docs/contribute.rst [hunk 5]
```diff
@@ -2,4 +2,31 @@
...
+This is an open-source project, and we welcome contributions from the community. 
+
+The Analytics Engine consists of two separate open-source repositories (hosted on GitHub) that you may contribute to.
+
+1. `climakitae <https://github.com/cal-adapt/climakitae>`_ (Climate Kit – Analytics Engine, get it?)
+
+	Building primarily on `xarray <https://docs.xarray.dev/en/stable/>`_ and the whole `Pangeo 	<https://pangeo.io/>`_ ecosystem, climakitae primarily offers tools that are:
+
+	* Meant to be run inside a Jupyter notebook (see below)
+	* Specific to the data for the `State of California <https://analytics.cal-adapt.org/data/>`_ that has 	been downscaled based on CMIP6 climate models.
```
Comment (vicford) [inline docs/contribute.rst:14]:
```suggestion
	* Specific to the downscaled CMIP6 climate model data produced for the `State of California <https://analytics.cal-adapt.org/data/>`_ .
```

File: docs/contribute.rst [hunk 6]
```diff
@@ -2,4 +2,31 @@
...
+
+2. The `cae-notebooks <https://github.com/cal-adapt/cae-notebooks>`_ repository 
+
+	`Jupyter <https://jupyter.org/>`_ notebooks in python which: 
+
+	* Highlight a particular climakitae tool, or
+	* Do a specific analysis for an example use-case or application of the data
```
Comment (vicford) [inline docs/contribute.rst:21]:
```suggestion
	* Perform a specific analysis for an example use-case or application of the data
```

File: docs/contribute.rst [hunk 7]
```diff
@@ -2,4 +2,31 @@
...
+
+We welcome contributions to either repository.
+
+Contributions to climakitae that we would welcome could involve a performance improvement, something which makes any of the tools more generally applicable, or adds a feature to handle a commonly-requested operation (without duplicating anything that already exists in the Pangeo ecosystem).
+
+Notebooks should be applications of general interest to our user community, which use the tools in climakitae whenever such a tool exists, and do not have dependencies outside of the `Pangeo libraries <https://github.com/pangeo-data/pangeo-docker-images/blob/master/pangeo-notebook/packages.txt>`_.
```
Comment (vicford) [inline docs/contribute.rst:27]:
```suggestion
Contributed notebooks should be applications of general interest to our user community, which use the tools in climakitae whenever such a tool exists, and do not have dependencies outside of the `Pangeo libraries <https://github.com/pangeo-data/pangeo-docker-images/blob/master/pangeo-notebook/packages.txt>`_.
```

File: docs/contribute.rst [hunk 8]
```diff
@@ -2,4 +2,31 @@
...
+
+If you have an idea for a contribution, please employ the following procedure:
```
Comment (vicford) [inline docs/contribute.rst:29]:
```suggestion
If you have an idea for contributing to either climakitae or cae-notebooks, please employ the following procedure:
```

File: docs/contribute.rst [hunk 9]
```diff
@@ -2,4 +2,31 @@
...
+
+1. Reach out to analytics@cal-adapt.org with a summary of what you are proposing to contribute to make sure that it is complementary to features we are already working on. 
+2. Submit a pull-request to the repository in question for members of our team to review.
```
Comment (vicford) [inline docs/contribute.rst:32]: Wondering if we also have set documentation etc. requirements that we also want to list here as well

Brainstorming an example below for discussion of how we want to structure contributions:

2a. For climakitae, new functions and features should be sufficiently documented via docstrings
2b. For contributed functionality, we recommend either integrating new code into our existing scripts (e.g., util/utils.py) or introducing new code into a new .py script in either the /tools or /utils folders
2c. Contributed notebooks to cae-notebooks should reside in a new folder housed in the /collaborative folder. We recommend utilizing a descriptive lowercase name for your contributed notebook. Contributed notebooks are requested to follow our notebook structure, consisting of: 
- Brief markdown overview/purpose of notebook, including a section titled "Intended Application" with a brief numbered list of the key take-away points a user will learn from the notebook
- Step 0: package imports
- Step 1: data retrieval and modifications if necessary
- Step 3+: analysis and optional figures
- Concluding Step: export of data object(s)
Our development team will assist with this structure, if you have questions.

**Review by vicford — CHANGES_REQUESTED** [review]
Thanks @gdicecco for taking this on! My last comment is more of do we have certain guidelines we want contributions to adhere to, rather than us formatting contributions as they come in.

File: docs/contribute.rst [hunk 10]
```diff
@@ -2,4 +2,41 @@
...
+
+Contributed notebooks should be applications of general interest to our user community, which use the tools in climakitae whenever such a tool exists, and do not have dependencies outside of the `Pangeo libraries <https://github.com/pangeo-data/pangeo-docker-images/blob/master/pangeo-notebook/packages.txt>`_.
+
+If you have an idea for contributing to either climakitae or cae-notebooks, please employ the following procedure:
+
+1. Reach out to analytics@cal-adapt.org with a summary of what you are proposing to contribute to make sure that it is complementary to features we are already working on. 
+2. Submit a pull-request to the repository in question for members of our team to review.
+
+	* For climakitae, new functions and features should be sufficiently documented via docstrings
+	* For contributed functionality, we recommend either integrating new code into our existing scripts (e.g., ``util/utils.py`` introducing new code into a new .py script in either the ``/tools`` or ``/utils`` folders
```
Comment (vicford) [inline docs/contribute.rst:35]:
```suggestion
	* For contributed functionality, we recommend either integrating new code into our existing scripts (e.g., ``util/utils.py``) or introducing new code into a new .py script in either the ``/tools`` or ``/utils`` folders
```

**Review by vicford — APPROVED** [review]
Just one small typo, otherwise looks great!

**Review by claalmve — APPROVED** [review]

**Pull request merged**